### ISSUE TITLE
Server stop when send push v0.50.1

### ISSUE THREAD

**Issue by Nemra1** [RocketChat/Rocket.Chat#5787]
Rocket.Chat Version: 0.50.1
Running Instances: 1
DB Replicaset OpLog:
Node Version:4.7.2
this happen now when trying send Push NTF
![2017-01-26_015309](https://cloud.githubusercontent.com/assets/22897987/22314535/bdb0d68e-e36a-11e6-8854-4dfee2995272.png)
Log
[34mI20170125-18:48:23.861(-5) Push: send message "@System" via query { userId: 'xudzgt2G8MSbvu3RX' } 
[34mI20170125-18:48:23.862(-5) Push: send to token { gcm: 'dBLnFeE5zRs:APA91bEgkEIPfZUKN5AvCP4vtJcrIyD_dohTMCmSFBVV_-o_CYw9cDKZ0T-q9FO3CUA0kmoY3kHfE0ZfD7xztz8pGe_Di093-Ml9-xEAD16SKZBOVkDz-MjmEiK-Dep1r-27QKfpYFym' } 
[34mI20170125-18:50:13.150(-5) Push: send message "@System" via query { userId: 'xudzgt2G8MSbvu3RX' } 
[34mI20170125-18:50:13.152(-5) Push: send to token { gcm: 'dBLnFeE5zRs:APA91bEgkEIPfZUKN5AvCP4vtJcrIyD_dohTMCmSFBVV_-o_CYw9cDKZ0T-q9FO3CUA0kmoY3kHfE0ZfD7xztz8pGe_Di093-Ml9-xEAD16SKZBOVkDz-MjmEiK-Dep1r-27QKfpYFym' }

**Comment by engelgabriel** [comment]
Can you provide the corret version number? 0.5.1 does not exists. Is it 0.50.1 or 0.51.0?

**Comment by Nemra1** [comment]
yup ...im sorry its 050.1

**Comment by engelgabriel** [comment]
Your logs shows no error, are you sure you are looking in the right place? Can you turn on the debug for the module?

**Comment by Nemra1** [comment]
my server restart after 15 mn now only after update  /so * im trying  to back to v0.4.4 again and check if this problem happen again or not ( evry thing is good after reback to older version with older DB BKUP )
NOW THIS SCREEN FROM CONSOLE FO V0.50.1
![2017-01-26_172319](https://cloud.githubusercontent.com/assets/22897987/22336840/3956d35c-e3ec-11e6-9fa9-b1eb5956ee6c.png)

**Comment by engelgabriel** [comment]
What do you have on your custom translations, seem to be a invalid formate, you must fix it.

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (113/100 (112%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext4.iad.github.net ([192.30.252.195]:59852 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.87)
	(envelope-from <noreply@github.com>)
	id 1cWo3k-003nRu-Np
	for greg@biomassiv.es; Thu, 26 Jan 2017 11:42:08 -0600
Date: Thu, 26 Jan 2017 09:41:27 -0800
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1485452487;
	bh=RLUjSr4/lxwQfXEsFqvjX04SHtLZSvyIXZf4LHlZpTY=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=KRglVJjs64CCBJHfWH1Rwl3xjio7rSYijeN2B5Xyq64HxBbjC5ad723Dm1IBL/Kxg
	 xT8RDAG3VeOK9vepIHoA8txNoGQp56/eFHXD1cOAnxAhdUwwBGg3eeFH1mKyk3+9T+
	 lTpfudH5N2M3kkI8g+YjvT+HM/fAKi6Eqw/K+N+g=

**Comment by Nemra1** [comment]
okey im remove it and ihave this now
WebSocket connection to 'wss://cdn.nemra-1.com/sockjs/478/ywev5acy/websocket' failed: Error during WebSocket handshake: Unexpected response code: 200
WebSocket connection to 'wss://cdn.nemra-1.com/sockjs/759/p2cr8ool/websocket' failed: Error during WebSocket handshake: Unexpected response code: 200
3e0464d….js?meteor_js_resource=true:216 CachedCollection private-settings => Init on login

**Comment by engelgabriel** [comment]
Are you trying to connect to the CDN?

**Comment by Nemra1** [comment]
No ..im setup rc server in this subdomain..cdn.nemra-1.com

**Comment by engelgabriel** [comment]
This error looks like an miss configuration. Looks like the domain cdn.nemra-1.com has a problem with wss:// connections.

**Comment by MartinSchoeler** [comment]
Since this looks like a configuration problem i will close this.

If you have any further questions please join our community server in the [#support](https://demo.rocket.chat/channel/support) channel or ask your question on stackoverflow with the [rocket.chat tag](https://stackoverflow.com/questions/tagged/rocket.chat) or [Server Fault](https://serverfault.com/search?q=Rocket.Chat)
